Cubical/Data/Unit/Base.agda

```diff
@@ -1,6 +1,11 @@
 {-# OPTIONS --cubical --no-import-sorts --safe #-}
 module Cubical.Data.Unit.Base where
+open import Cubical.Foundations.Prelude
 
 -- Obtain Unit
 open import Agda.Builtin.Unit public
   renaming ( ⊤ to Unit )
+
+-- Lifted version
+data Unit* {ℓ : Level} : Type ℓ where
```

Comment:
Maybe it would be better if this would be the default Unit? Also the commented is strange, why not "universe polymorphic version"?

Reply:
Might be better to define it using the existing `Lift` and `Unit`:
```
Unit* : {ℓ : Level} → Type ℓ
Unit* = Lift Unit

pattern tt* = lift tt
```
I suspect that this would be inconvenient to use as the default `Unit`. For one thing, you're often going to need to specify the level explicitly, `Unit* {ℓ}`, and you'll need more level quantification everywhere.

Reply:
Ah ok, I agree with @ecavallo 's suggestion then!

Reply:
Right, the lifted one is probably better to use with truncations since it inherits definitional equality from Unit. The reason I defined it like this is that I didn't want the definitional equality (I "lock" theorems in ZCohomology by giving them with respect to an element in Unit* and that doesn't work if you have definitional equality). I think I'll add a third Unit type called "lockUnit" or something then.

Reply:
Others have used the name `Block` for a unit type used like this. Even giving it a phantom string parameter, `Block : String -> Type` so the `String` can document what you're blocking.